### ISSUE TITLE
Cultural vacate lists

### ISSUE THREAD

**Issue by RustyGray** [clld/vanuatuvoices#6]
Can we discuss the best we to add these please?

**Comment by maryewal** [comment]
Aviva's "culture list" is different from what was collected since. In 2018 and 2019, 
I did kinship paradigms and an "extended vocab" list that included additional numerals 10-20 in addition to environmentally relevant and culturally relevant vocab. Much of the original culture list was super specific to Malekula did not work at all once we tried eliciting it elsewhere.

Could we have some sort of "tab" display? Where we can select basic vocab, kinship, numerals, and other categories?  I think this would be a good way of exhibiting the data and making sense of the different lists that were collected. We might also consider island-specific "tabs" too - terms for nambas, for example, are only found in Malekula. Not a useful comparison cross-Vanuatu, but surely interesting to see variation within Malekula.

**Comment by RustyGray** [comment]
PS.  Is there data come from Epi and Emae?
 R.

Russell Gray
Director, Max Planck Institute for the Evolutionary Anthropology
Head of the Department of Linguistic and Cultural Evolution
TEL: +49-3641-68 68 01
FAX: +49-3641-68 68 68
Assistant schueck@shh.mpg.de<mailto:schueck@shh.mpg.de>
http://www.shh.mpg.de/2375/en
http://language.psy.auckland.ac.nz/
http://scholar.google.co.nz/citations?user=FzE-34YAAAAJ&hl=en



On 24. Nov 2020, at 12:47, Mary Walworth <notifications@github.com<mailto:notifications@github.com>> wrote:



Aviva's "culture list" is different from what was collected since. In 2018 and 2019,
I did kinship paradigms and an "extended vocab" list that included additional numerals 10-20 in addition to environmentally relevant and culturally relevant vocab. Much of the original culture list was super specific to Malekula did not work at all once we tried eliciting it elsewhere.

Could we have some sort of "tab" display? Where we can select basic vocab, kinship, numerals, and other categories? I think this would be a good way of exhibiting the data and making sense of the different lists that were collected. We might also consider island-specific "tabs" too - terms for nambas, for example, are only found in Malekula. Not a useful comparison cross-Vanuatu, but surely interesting to see variation within Malekula.

**Comment by maryewal** [comment]
Yes, definitely. It is all transcribed, we just have to see if Darja will have time to post-process and get it all marked up. Starting with Lana's 2018 recordings and working our way up.

**Comment by xrotwang** [comment]
@maryewal see https://github.com/lexibank/vanuatuvoices/issues/6
